src/main/java/http_server/Response.java

```diff
@@ -1,97 +1,48 @@
 package http_server;
 
-import java.time.LocalDateTime;
-import java.time.ZoneId;
-import java.time.ZonedDateTime;
-import java.time.format.DateTimeFormatter;
 import java.util.LinkedHashMap;
-import java.util.Map;
 
 public class Response {
-    RequestParser request;
-    Router router;
-    LinkedHashMap<String, String> headerCollection = new LinkedHashMap<>();
-    String status;
-    String responseBody = "";
-    String crlf = "\r\n";
+    private ResponseBuilder responseBuilder;
+    private LinkedHashMap<String, String> headerCollection = new LinkedHashMap<>();
+    private String status;
+    private String responseBody = "";
 
-    public Response(RequestParser requestParser, Router router) {
-        System.err.println(router);
-        this.request = requestParser;
-        this.router = router;
-        this.status = "200 OK";
+    public Response(Request request, Router router) {
+        this.responseBuilder = new ResponseBuilder(this, request, router);
```

Comment:
It seems odd to me that `ResponseBuilder` is a dependency here. I'd expect this to be inverted so that ResponseBuilder is responsible for building up this response object.

Reply:
Will handle this in the next PR -- I'm having some trouble thinking of how to invert the dependency